Cargo.toml

```diff
@@ -66,7 +66,7 @@ serde = { version = "1", default-features = false, features = [
   "std",
 ] }
 snafu = { version = "0.8", default-features = false, features = ["std"] }
-tokio = { version = "1", default-features = false }
+tokio = { version = "1", default-features = false, features = ["rt", "rt-multi-thread", "io-util", "macros"] }
```

Comment:
Hmmm, do we need this here? I see we're adding these features in `lib/saluki-common/Cargo.toml` directly.

(Ignoring that I don't think the scrubber actually needs to be `async` and so we might not need these features at all for `saluki-common`.)